catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -70,9 +70,42 @@ function (ich,Marionette,_,$,Q,ModalConfig,ModalRegistry,EmptyView,wreqr,Utils,d
             this.model.set('available', this.statusModel.get('value'));
             this.render();
         },
+        //function to called on every RegistryRow when the refresh button is pressed
         updateRemoteStatus: function() {
-            if(this.remoteStatusModel.get('value')){
-                this.model.set('remoteName', this.remoteStatusModel.get('value').remoteName);
+            //collection of configurations which has all models representing every entry in the registryRows.
+            var allConfigurations = this.model.collection.models;
+
+            //ConfigIds has values that are all unique since they cant be duplicated locally (there is a dup check on creation locally)
+            // (only can be dupicated whena remote uses a name that is already used locally
+            var configIds = {};
+            for(var i =0; i<allConfigurations.length; i++){
+                if(!configIds[allConfigurations[i].get('attributes').id]) {
+                    configIds[allConfigurations[i].get('attributes').id] = "";
+                }
+                // else{
```

Comment:
Remove this commented out code. The backend will ensure that there are no duplicate names